UefiPayloadPkg/SPI/BlSMMStoreDxe.h

```diff
@@ -0,0 +1,118 @@
+/** @file  BlSMMStoreDxe.h
```

Comment:
This file name needs change for one related to AMD SPI

Reply:
@IgorBagnucki

Reply:
File is deleted now